### DISCUSSION TITLE
Add Cyborg and AI playable characters

### DISCUSSION THREAD

**Discussion by MrCredible** [space-wizards/space-station-14#18400] [Feature Request]
EDIT:
Since there is already a pull request for cyborgs https://github.com/space-wizards/space-station-14/pull/18136 , then this feature request is for an AI as head of the silicon department.

Are there any plans to add cyborgs and AI?
My opinion is that the more opportunities there are to role play with different rulesets, the better.
This is why playing antag is so much fun.

Adding cyborg and AI roles would increase these opportunities and allow players a less random chance to have a different ruleset (Asimov Laws) to abide by.

Cyborgs and AI can have the opportunity to be taken over by an antag and AI can have an opportunity to roll as an antag.

For reference, I am basing my request upon how TG server in space station 13 implemented their cyborg and AI.
https://tgstation13.org/wiki/Cyborg
https://tgstation13.org/wiki/AI

Reference to malf AI:
https://tgstation13.org/wiki/Guide_to_malfunction

**Comment by moonheart08** [comment]
You could've just quickly searched through the open pull requests.

**Reply by MrCredible** [reply]
I did not find anything in issues or discussions.

As well, that pull request only adds cyborgs and not AI.

For reference:
https://github.com/space-wizards/space-station-14/pull/18136

So I will leave this discussion open.

**Reply by deltanedas** [reply]
takes time to implement things big shock

**Reply by Jark255** [reply]
Found some kind of plans in #6923. There are already a lot of those parts in-place (binary radio, cameras, MMIs/posibrains) even silicon laws on top of that. But I still didn't understand some things and haven't found in the docs the design document (except the borgs one).

How the player will be able to see, considering the rejection from ss13-style "floating eye" movement? It will be a window with station minimap while the default view is from the AI Core or something entirely else?

And the door remote as intrinsic will be like some console in aghost since, again, the "floating eye" is dropped or there were some other thoughts? And by the way, how other machines will be (if they would be) controlled remotely? Like intercoms or air alarms?

**Reply by IcedQuinn** [reply]
Floating-eye is the most ergonomic from a player perspective. Before that 13 made you click a camera name from a list or cycle through them one-by-one. There could be odd issues with coding it though idk
